.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -23,30 +23,30 @@ with the release so please keep this issue up to date:
   release number to the GitHub tag page the first time a given release number
   appears in the checkboxes.
 - Add any updates that may be of interest to the community (such as delays) as
-  comments on this issue, including after the release is pushed to production.
+  comments on this issue, including after the release is promoted to Stable.
 - Keep the title of the issue updated to reflect whether this issue is tracking
-  the Canary or the build in Production.
+  the Experimental/Beta builds or the build in Stable.
 
 Note: remove the backticks (``) from the link.
 -->
 
 - [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
-- [ ] Release <RELEASE_NUMBER> pushed to dev channel (<CL submit time>)
-- [ ] Release <RELEASE_NUMBER> pushed to 1% (<CL submit time>)
-- [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (opt-in) channels (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (1% traffic) channels (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to Stable channel(<CL submit time>)
```

Comment:
nit: For consistency, add a space before the open parenthesis.

Reply:
Nice catch; done.